tests/api.spec.ts

```diff
@@ -194,20 +194,24 @@ describe('Requests to public API', () => {
   });
 
   it('request analysis with missing files', async () => {
-    const response = await getAnalysis({
-      baseURL,
-      sessionToken,
-      bundleId: fakeBundleIdFull,
-      includeLint: false,
-      severity: 1,
-    });
+    let response;
+    do {
+      response = await getAnalysis({
+        baseURL,
+        sessionToken,
+        bundleId: fakeBundleIdFull,
+        includeLint: false,
+        severity: 1,
+      });
+    } while (response.type === 'success');
```

Comment:
I don't understand. Why should getAnalysis return 'success' for incomplete bundles? If this request loops once, it may be stuck forever until the bundle expires (1 day of test is not feasible).

Reply:
Because in worst case, this request will have to wait to be added to the queue and we will not know if it's failed or not, so it will return success response.

Reply:
this is a test. Tests have timeouts themselves and will not pass in this case

Reply:
I don't know if the test timeout is a valid pattern for this scenario, but if you're sure it applies I'll resolve the conversation for now.